MyWeightTests/UITableViewExtensionsTests.swift

```diff
@@ -0,0 +1,162 @@
+//
+//  UITableViewExtensionsTests.swift
+//  MyWeight
+//
+//  Created by Bruno Koga on 22/06/16.
+//  Copyright © 2016 Diogo Tridapalli. All rights reserved.
+//
+
+import XCTest
+@testable import MyWeight
+
+class InvarianteTableViewCell: UITableViewCell { }
+class EmojiTableViewCell😎: UITableViewCell { }
+class a0_$WÅ´Ø: UITableViewCell { }
```

Comment:
![How I imagine the mods of most subs when this election is finally over](http://i.imgur.com/g5LoDk8.gif)
It's just a test...